indices/services.py

```
@@ -0,0 +1,53 @@
+        }
+        
+        params = {
+            "FID_COND_MRKT_DIV_CODE": "U",
+            "FID_INPUT_ISCD": iscd,
+            "FID_INPUT_DATE_1": start_date,
+            "FID_INPUT_DATE_2": end_date,
+            "FID_PERIOD_DIV_CODE": "D",
+            "FID_ORG_ADJ_PRC": "0"
+        }
+        
+        response = requests.get(url, headers=headers, params=params)
```

Ruff (0.14.11)

Probable use of `requests` call without timeout

(S113)